specification/storagepool/resource-manager/Microsoft.StoragePool/stable/2021-08-01/storagepool.json

```diff
@@ -0,0 +1,2092 @@
+          "type": "array",
+          "items": {
+            "$ref": "#/definitions/EndpointDependency"
+          }
+        }
+      }
+    },
+    "EndpointDependency": {
+      "description": "A domain name that a service is reached at, including details of the current connection status.",
+      "type": "object",
+      "properties": {
+        "domainName": {
+          "description": "The domain name of the dependency.",
+          "type": "string"
+        },
+        "endpointDetails": {
+          "description": "The IP Addresses and Ports used when connecting to DomainName.",
+          "type": "array",
+          "items": {
+            "$ref": "#/definitions/EndpointDetail"
+          }
+        }
+      }
+    },
+    "EndpointDetail": {
+      "description": "Current TCP connectivity information from the App Service Environment to a single endpoint.",
+      "type": "object",
+      "properties": {
+        "ipAddress": {
+          "description": "An IP Address that Domain Name currently resolves to.",
+          "type": "string"
+        },
+        "port": {
+          "format": "int32",
+          "description": "The port an endpoint is connected to.",
+          "type": "integer"
+        },
+        "latency": {
+          "format": "double",
+          "description": "The time in milliseconds it takes for a TCP connection to be created from the App Service Environment to this IpAddress at this Port.",
+          "type": "number"
+        },
+        "isAccessible": {
+          "description": "Whether it is possible to create a TCP connection from the App Service Environment to this IpAddress at this Port.",
+          "type": "boolean"
+        }
+      }
+    },
+    "DiskPoolListResult": {
+      "type": "object",
+      "required": [
+        "value"
+      ],
+      "description": "List of Disk Pools",
+      "additionalProperties": false,
+      "properties": {
+        "value": {
+          "description": "An array of Disk pool objects.",
+          "type": "array",
+          "items": {
+            "$ref": "#/definitions/DiskPool"
+          }
+        },
+        "nextLink": {
+          "description": "URI to fetch the next section of the paginated response.",
+          "type": "string",
+          "readOnly": true
+        }
+      }
+    },
+    "ResourceSkuListResult": {
+      "type": "object",
+      "properties": {
+        "value": {
+          "type": "array",
+          "items": {
+            "$ref": "#/definitions/ResourceSkuInfo"
+          },
+          "description": "The list of StoragePool resource skus."
+        },
+        "nextLink": {
+          "description": "URI to fetch the next section of the paginated response.",
+          "type": "string"
+        }
+      },
+      "description": "List Disk Pool skus operation response."
+    },
+    "ResourceSkuInfo": {
+      "type": "object",
+      "description": "Resource SKU Details",
+      "properties": {
+        "resourceType": {
```

Comment:
>resourceType [](http://example.com/codeflow?start=9&length=12)

You might want to annotate this property as readOnly: true, make sure to do that for all read only properties.